### PR TITLE
CORDA-822: Fix-up "customOverrides" typo.

### PR DESCRIPTION
**Pull request by vkolomeyko**

### CODE REVIEW

**Comment by josecoll** [comment]
Probably better to clean this up - I was experimenting when testing Jolokia.

2 changes:
`val alice = startNode(providedName = ALICE.name, rpcUsers = listOf(user), customOverrides = mapOf("devMode" to "true"))`
becomes
`val alice = startNode(providedName = ALICE.name, rpcUsers = listOf(user))`

remove ` "" to "true"))`

**Comment by vkolomeyko** [comment]
So, do you want me to revert your changes in this file completely?

**Comment by josecoll** [comment]
Yes please. My mistake for leaving them in.

File: tools/explorer/src/main/kotlin/net/corda/explorer/ExplorerSimulation.kt [hunk 1]
```diff
@@ -65,14 +64,14 @@ class ExplorerSimulation(private val options: OptionSet) {
 
     private fun startDemoNodes() {
         val portAllocation = PortAllocation.Incremental(20000)
-        driver(portAllocation = portAllocation, extraCordappPackagesToScan = listOf("net.corda.finance"), waitForAllNodesToFinish = true, jmxPolicy = JmxPolicy(true)) {
```
Comment (josecoll) [inline tools/explorer/src/main/kotlin/net/corda/explorer/ExplorerSimulation.kt]: should stay.

File: tools/explorer/src/main/kotlin/net/corda/explorer/ExplorerSimulation.kt [hunk 2]
```diff
@@ -24,7 +24,6 @@ import net.corda.node.services.Permissions.Companion.startFlow
 import net.corda.nodeapi.internal.config.User
 import net.corda.testing.ALICE
 import net.corda.testing.BOB
-import net.corda.testing.driver.JmxPolicy
```
Comment (josecoll) [inline tools/explorer/src/main/kotlin/net/corda/explorer/ExplorerSimulation.kt]: should stay.

**Comment by vkolomeyko** [comment]
Should be done now.
@josecoll - please review and approve.

**Pull request merged**